src/js/components/PodInstancesTable.js

```diff
@@ -37,6 +40,24 @@ class PodInstancesTable extends React.Component {
     });
   }
 
+  componentWillMount() {
```

Comment:
Normally we use the StoreMixin instead of having these three methods, like so: https://github.com/dcos/dcos-ui/blob/master/src/js/components/ServiceDetailTaskTab.js#L15

But, seeing as we need to get this in, and it is bound change soon anyways, let's not worry about it for now